wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wdl/WdlWriter.scala

```diff
@@ -0,0 +1,450 @@
+package wdl.draft3.transforms.wdlom2wdl
+
+import simulacrum.typeclass
+import wdl.model.draft3.elements.CommandPartElement.{PlaceholderCommandPartElement, StringCommandPartElement}
+import wdl.model.draft3.elements.ExpressionElement._
+import wdl.model.draft3.elements.MetaValueElement._
+import wdl.model.draft3.elements._
+import wom.types._
+
+import scala.language.implicitConversions
+
+@typeclass
+trait WdlWriter[A] {
+  def toWdl(a: A): String
+
+  // Stolen from WomGraph.scala
+  def indent(s: String) = s.lines.map(x => s"  $x").mkString(System.lineSeparator)
+  def combine(ss: Iterable[String]) = ss.mkString(start="", sep=System.lineSeparator, end=System.lineSeparator)
+  def indentAndCombine(ss: Iterable[String]) = combine(ss.map(indent))
+}
+
+object WdlWriter {
```

Comment:
I *think* I'd prefer to have this implementation in a different file to the typeclass, to make it clear that "here is a typeclass that anyone can implement" and "here is one implementation"